protoc-gen-twirp_ruby/main.go

```diff
@@ -217,6 +219,16 @@ func writeResponse(w io.Writer, resp *plugin.CodeGeneratorResponse) {
 	}
 }
 
+// Modules converts protobuf package name to a list of Ruby module names to
+// represent it.
+func Modules(pkgName string) []string {
+	var parts []string
+	for _, p := range strings.Split(pkgName, ".") {
+		parts = append(parts, CamelCase(p))
+	}
+	return parts
+}
```

Comment:
The name "Modules" is a little bit generic, even for a small file like this. Rename it to something like `PackageToModules` or `GetRubyModules`. The comment can also have an example: `e.g. "my.package" => {"My", "Package"}`.